### ISSUE TITLE
be able to specify full path to python

### ISSUE THREAD

**Issue by zbeekman** [ParaToolsInc/taucmdr#316]
We need to be able to specify the full path to python

**Comment by khsa1** [comment]
Is this something that can be specified to tau_python? And does TAU need to be built with this version of Python in order to run the code?

**Comment by nchaimov** [comment]
`tau_python` takes an argument `-tau-python-interpreter=` to specify the path to an interpreter. TAU does need to be built against the specific version of Python used at runtime. So I think TAU Commander will need to query the version of the specified interpreter and incorporate that into the hash for the TAU version

**Comment by khsa1** [comment]
Thanks, I'll work on implementing that.

I tested with just TAU and ran into one issue. If I specify the `-tau-python-interpreter` to be `python3`, then I get:
> module 'tau' not found in PYTHONPATH

**Comment by sameershende** [comment]
Hi Sam,
  Is this error with git head tau2 with tau_python?
  Thanks,
  - Sameer

**Comment by nchaimov** [comment]
Did you build TAU with a Python 3 version? That is the error you will get if you try to use a Python 2 module with Python 3 or vice versa.

**Comment by khsa1** [comment]
Yes, this was with git head tau2. The issue was that I built TAU with `python2` but tried to use `python3`.
